tests/Feature/GraphQL/Queries/GetCollectionsTest.php

```diff
@@ -45,6 +47,32 @@ public function test_it_can_fetch_with_no_args(): void
         $this->assertTrue(count($response['edges']) > 0);
     }
 
+    public function test_it_can_get_total_count_correctly(): void
+    {
+        Schema::disableForeignKeyConstraints();
+        Collection::truncate();
+        Token::truncate();
+        CollectionAccount::truncate();
+        Schema::enableForeignKeyConstraints();
+
+        $collections = Collection::factory(2)->create();
+        $collection1 = $collections->shift();
+        Token::factory($tokenCount1 = random_int(1, 100))->create(['collection_id' => $collection1->id]);
+        CollectionAccount::factory($accountCount1 = random_int(1, 100))->create(['collection_id' => $collection1->id]);
+        $collection2 = $collections->shift();
+        Token::factory($tokenCount2 = random_int(1, 100))->create(['collection_id' => $collection2->id]);
+        CollectionAccount::factory($accountCount2 = random_int(1, 100))->create(['collection_id' => $collection2->id]);
```

Comment:
```suggestion
        Token::factory($tokenCount1 = random_int(16, 100))->create(['collection_id' => $collection1->id]);
        CollectionAccount::factory($accountCount1 = random_int(16, 100))->create(['collection_id' => $collection1->id]);
        $collection2 = $collections->shift();
        Token::factory($tokenCount2 = random_int(16, 100))->create(['collection_id' => $collection2->id]);
        CollectionAccount::factory($accountCount2 = random_int(16, 100))->create(['collection_id' => $collection2->id]);
```
Let's use 16 as the min value seeing as the UI was happy to display up to 15 without issue, so this makes sure we don't get false positives for 1-15 which looks like would always work.